src/app/pubspec.yaml

```diff
@@ -28,6 +28,9 @@ dependencies:
   path_provider: ^2.1.3
   share_plus: ^9.0.0
   alice: ^0.4.1
+  flutter_localizations:
+    sdk: flutter
+  intl: any
```

Comment:
why the any keyword here?

Reply:
https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization
Because they say so